ftl/README.md

```diff
@@ -28,18 +28,33 @@ o/my-project/cloudcats-node-app:latest
 ```
 
 ## Releases
-Currently FTL is released in .par format for each supported runtime.
-node:
-[FTL Node par (latest)](https://storage.googleapis.com/gcp-container-tools/ftl/node/latest/ftl.par)
-Specific version (based on git $COMMIT_SHA)`https://storage.googleapis.com/gcp-container-tools/ftl/node/$COMMIT_SHA/ftl.par`
+Currently FTL is released in .par format for each supported runtime.  The latest release is v0.2.0, changelog [here](https://github.com/GoogleCloudPlatform/runtimes-common/blob/master/ftl/CHANGELOG.md)
 
-python:
-[FTL Python par (latest)](https://storage.googleapis.com/gcp-container-tools/ftl/python/latest/ftl.par)
-Specific version (based on git $COMMIT_SHA)`https://storage.googleapis.com/gcp-container-tools/ftl/python/$COMMIT_SHA/ftl.par`
+###node
 
-php:
-[FTL PHP par (latest)](https://storage.googleapis.com/gcp-container-tools/ftl/php/latest/ftl.par)
-Specific version (based on git $COMMIT_SHA)`https://storage.googleapis.com/gcp-container-tools/ftl/php/$COMMIT_SHA/ftl.par`
+[v0.2.0](https://storage.googleapis.com/gcp-container-tools/ftl/node/node-v0.2.0/ftl.par)
+
+[HEAD](https://storage.googleapis.com/gcp-container-tools/ftl/node/latest/ftl.par)
+
+Specific version (based on git $COMMIT_SHA)
+`https://storage.googleapis.com/gcp-container-tools/ftl/node/$COMMIT_SHA/ftl.par`
+
+###python
```

Comment:
Think you might need a space here `### python` for the heading

Reply:
done